### PR TITLE
Pract.04: Add 01-03 | Momchil Uzunov

### PR DESCRIPTION
**Pull request by momouzunov2003**
Momchil Uzunov
6 1MI0600146

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task 03.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+bool isPositive(int num)
+{
+    if (num > 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 03.cpp:5]: num > 0 is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task 03.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    else 
+    { 
+        return false;
+    }
+}
+
+bool isNegative(int num)
+{
+    if (num < 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 03.cpp:17]: num > 0 is a boolean expression, so it returns a boolean value that you can directly return

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Some refactoring needed

**Review by momouzunov2003 — COMMENTED** [review]
Fixed Task03

File: SI/Sem.04/Pract.04/Task 03.cpp [hunk 3]
```diff
@@ -2,26 +2,12 @@
...
 {
-    if (num > 0)
-    {
-        return true;
-    }
-    else 
-    { 
-        return false;
-    }
+   return (num>0);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task 03.cpp:5]: return num > 0;

No brackets and spaces between variable & operations

File: SI/Sem.04/Pract.04/Task 03.cpp [hunk 4]
```diff
@@ -2,26 +2,12 @@
...
 {
-    if (num < 0)
-    {
-        return true;
-    }
-    else
-    {
-        return false;
-    }
+    return (num<0);
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/Task 03.cpp:10]: Same as IsPositive func comment.

**Review by Justsvetoslavov — APPROVED** [review]

**Comment by Justsvetoslavov** [comment]
Closing due to merged solutions.
Address comments and contact us if you have questions.

**Pull request closed without merging**